workflow/snakemake_rules/main_workflow.smk

```diff
@@ -93,18 +92,15 @@ rule align:
             --sequences {input.sequences} \
             --strip-prefixes {params.strain_prefixes:q} \
             --output /dev/stdout 2> {params.sanitize_log} \
-            | nextalign \
+            | nextalign2 run \
             --jobs={threads} \
             --reference {input.reference} \
             --genemap {input.genemap} \
-            --genes {params.genes} \
-            --sequences /dev/stdin \
-            --output-dir {params.outdir} \
-            --output-basename {params.basename} \
+            --output-translations {params.output_translations} \
             --output-fasta {params.uncompressed_alignment} \
```

Comment:
According to [the changelog](https://github.com/nextstrain/nextclade/commit/7ed87ffe335e0cef92f43a17372bde40693b7e88), version 2 of the Nextalign CLI can write compressed output files, perhaps rendering the `uncompressed_alignment` parameter unnecessary.